netbox/netbox/settings.py

```diff
@@ -586,9 +586,10 @@ def _setting(name, default=None):
     "LICENSE": {"name": "Apache v2 License"},
     "VERSION": VERSION,
     'COMPONENT_SPLIT_REQUEST': True,
+    'REDOC_DIST': 'SIDECAR',
```

Comment:
Can you clean this up so that the dictionary keys and values use either single or double quotes consistently please?

Reply:
done